README.md

```diff
@@ -866,6 +866,7 @@ See also <https://en.wikipedia.org/wiki/List_of_streaming_media_systems>, <https
 - [DomainMOD](https://domainmod.org) - Application to manage your domains and other internet assets in a central location. DomainMOD includes a Data Warehouse framework that allows you to import your WHM/cPanel web server data so that you can view, export, and report on your data. ([Demo](https://demo.domainmod.org), [Source Code](https://github.com/domainmod/domainmod)) `GPL-3.0` `PHP`
 - [EasyBook Project](http://easybook-project.org/) - Book publishing as easy as it should be. ([Source Code](https://github.com/javiereguiluz/easybook)) `MIT` `PHP`
 - [Flox](https://github.com/devfake/flox) `⚠` - Self hosted movie, TV series and anime watch list with a 3-point rating system. Uses The Movie Database backend for information. ([Demo](http://80.240.132.120/flox-demo/public/)) `MIT` `PHP`
+- [formspree](https://formspree.io/) - Just send your form to our URL and we'll forward it to your email. No PHP, Javascript or sign up required. ([Demo](http://testformspree.com/), [Source Code](https://github.com/formspree/formspree)) `AGPL-3.0` `Python`
```

Comment:
> Formspree requires a PostgreSQL database and uses SendGrid to send emails

@bysslord can you add the symbol since sends mail via a third-party the user doesn't control? Like so `[formspree](https://formspree.io/) `⚠` - Just send your...`

Thanks